syz-manager/manager.go

```diff
@@ -1072,6 +1067,11 @@ func (mgr *Manager) fuzzerConnect() ([]rpctype.RPCInput, BugFrames, bool) {
 func (mgr *Manager) machineChecked(a *rpctype.CheckArgs, enabledSyscalls map[*prog.Syscall]bool) {
 	mgr.mu.Lock()
 	defer mgr.mu.Unlock()
+	var err error
+	mgr.coverFilterFilename, mgr.coverFilter, err = createCoverageFilter(mgr.cfg)
```

Comment:
I think this won't work unfortunately, if coverage filter enabled we should have been copied the filter _before_ we started the VM. 
This will either crash in executor when it won't find the coverage bitmap file, or first batch of VMs will ignore the coverage filter.
We somehow need to move copying of the coverage filter after this point. Earlier I suggested to send it to the fuller in Machine.Check reply, I think it should work.

You can test coverage filter with something simple as:
```
	"cover_filter": {
		"files": ["net/"]
	}
```

Reply:
Fix and tested.